src/main/fc/rc_controls.c

```diff
@@ -82,11 +82,12 @@ PG_RESET_TEMPLATE(rcControlsConfig_t, rcControlsConfig,
     .yaw_control_reversed = false,
 );
 
-PG_REGISTER_WITH_RESET_TEMPLATE(armingConfig_t, armingConfig, PG_ARMING_CONFIG, 1);
+PG_REGISTER_WITH_RESET_TEMPLATE(armingConfig_t, armingConfig, PG_ARMING_CONFIG, 2);
 
 PG_RESET_TEMPLATE(armingConfig_t, armingConfig,
     .gyro_cal_on_first_arm = 0,
-    .auto_disarm_delay = 5
+    .auto_disarm_delay = 5,
+    .prearm_until_first_arm = 0
```

Comment:
`= false` would be clearer.

Reply:
Hi Steve! Wrote `= 0` by analogy with other [ ON | OFF ] cli values. Just used the style of `gyro_cal_on_first_arm` and similar. They all with 0

Reply:
See above.